src/share/classes/com/sun/javatest/regtest/exec/Agent.java

```diff
@@ -927,6 +986,8 @@ private static String getKey(File dir, JDK jdk, List<String> vmOpts) {
         private float timeoutFactor = 1.0f;
         private int maxPoolSize;
         private Duration idleTimeout;
+        // default is 2 i.e. we retry a failed agent selection once
+        private int numAgentSelectionAttempts = 2;
```

Comment:
The limit of 2 is set here but also later on.

Reply:
I did not understand this review comment. With the updated changes in this PR, the default value now is set to 1, here too. Did you mean something more?